bike-erp/backend/src/main/services/emailService/emailService.ts

```diff
@@ -0,0 +1,49 @@
+import nodemailer from "nodemailer";
+import { EmailDAO } from "../../dao/EmailDAO";
+
+export class EmailService {
+
+    private static emailService: EmailService | undefined;
+    private static emailDAO: EmailDAO = new EmailDAO();
+
+    private constructor() {}
+
+    // Instanciating singleton
+    public static getEmailService() {
+      if (this.emailService === undefined) {
+        this.emailService = new EmailService();
+      } else {
+        return this.emailService;
+      }
+    }
+    
+    public static async email(emailAddress:string, emailSubject : string, emailBody: string) {
+        // create reusable transporter object using the default SMTP transport
+        const transporter = nodemailer.createTransport({
+        host: "smtp.gmail.com",
+        port: 587,
+        secure: false, // true for 465, false for other ports
+        auth: {
+            user: 'bikekinginc@gmail.com',
+            pass: 'emailpassword_123',
+        },
+        tls: {
+          rejectUnauthorized: false
+      }
```

Comment:
formating fix here for the curly bracket and the end of method

Reply:
done!